larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx

```diff
@@ -66,12 +67,13 @@ namespace opdet {
                pmtana::pulse_param const& pulse,
                std::vector<recob::OpHit>& hitVector,
                detinfo::DetectorClocksData const& clocksData,
-               calib::IPhotonCalibrator const& calibrator)
+               calib::IPhotonCalibrator const& calibrator,
+	       bool use_start_time)
   {
 
     if (pulse.peak < hitThreshold) return;
 
-    double absTime = timeStamp + pulse.t_max * clocksData.OpticalClock().TickPeriod();
+    double absTime = timeStamp + clocksData.OpticalClock().TickPeriod() * (use_start_time ? pulse.t_start : pulse.t_end);
```

Comment:
Shouldn't the backwards-compatible change be:
```diff
double absTime = timeStamp + clocksData.OpticalClock().TickPeriod() *
- (use_start_time ? pulse.t_start : pulse.t_end);
+ (use_start_time ? pulse.t_start : pulse.t_max);
```

Reply:
ah yes! Sorry for this mistake in rush.

Reply:
No problem, Kazu.